### ISSUE TITLE
Code Updated, Mask Annotations and Open-Vocabulary meaning

### ISSUE THREAD

**Issue by PhucNDA** [OpenSun3D/OpenSun3D.github.io#2]
Hi, thanks for a wonderful challenge.

I have 3 main problems:

1. I want to express my concern about the code last update. I am looking forward to participating to this challenge. The [official OpenSun3D page](https://opensun3d.github.io/) informs that the Dev phase has already begun, but the code is not fully-updated.

2. Furthermore, I follow the download instruction provided but don't see any annotated mask GTs in the dev-set. Correct me if I am wrong or have I missed any important information?
 
3. Lastly, I personally find that the workshop shares some similarity with this [languagefor3dscene](https://languagefor3dscenes.github.io/ECCV2022/) apart from 3D open-vocabulary (OV) instances. What is the difference in queries between the challenges? (is OV query in this challenge shorter and simpler by omitting subject, verbs, objects; without spatial relation/with simple spatial relation, etc.). As I discover the queries in this challenge is totally shorter and less comprehensive than the languagefor3dscene challenge `/queries_development_scenes.csv.`  Please clarify about the setting, it helps the challenge much more approachable.

Once again, thanks for the challenge, I really appreciate the effort of the development team.
Looking forward to the clarification.
PhucNDA.

**Comment by aycatakmaz** [comment]
Hi @PhucNDA, 

Thanks a lot for your interest!

1. Thanks for bringing this to our attention, we have updated the dates on the website. The Dev phase started on July 13th, as data download instructions and submission instructions are already available. The submission portal will be opened to public in a few days. Please note that the submission portal does not have any additional instructions that are not already provided in the [README.md](https://github.com/OpenSun3D/OpenSun3D.github.io/blob/main/challenge/README.md). In the meantime, to locally evaluate your predictions, you can use the [demo evaluation script](https://github.com/OpenSun3D/OpenSun3D.github.io/blob/main/challenge/demo_eval.py) we provide.

2. You are correct, we did not initially release the annotated GT masks for the Dev scenes. However, in order to clarify the task and the type of annotations, we have just added them to our repository. You can find the GT annotations [here](https://github.com/OpenSun3D/OpenSun3D.github.io/tree/main/challenge/benchmark_data/gt_development_scenes), and an explanation about the GT data format [here](https://github.com/OpenSun3D/OpenSun3D.github.io/blob/main/challenge/benchmark_data/gt_data_format.md). Hope this helps!

3. While there are similarities, as we also have text-based queries describing objects in a 3D scene, there are two main differences. First of all, we focus on the dense 3D object instance segmentation - only localizing the object by predicting its bounding box is not the focus of our challenge. We instead expect a per-point segmentation of multiple objects. For example, there might be multiple objects that fit the given query. Our task is to identify all of these instances that fit the description, by proposing instance masks over the point-cloud, describing which points in the scene correspond to these instances, and which points correspond to the background. Secondly, our queries are free-form open vocabulary descriptions that do not necessarily follow strong rules in terms of length or query sentence structure. That being said, in our challenge, we would like to highlight novel/uncommon object categories that are often not represented in existing annotated 3D indoor datasets for instance segmentation. In the Test phase, you can expect more queries with short object property descriptions (object type, color, material, affordance etc.), however there will also be some queries describing spatial relations. Hope this clarifies the setting a bit more!

If you have any other questions, let us know!

Best,
Ayca

**Comment by PhucNDA** [comment]
Hi again, thanks for the detailed clarification.

After our team discussion, we have many concerns about training data and published pretrained model.

As the challenge stated: "For this challenge, in order to develop or train your model, you can only use the scenes provided in the ARKitScenes Training set or the Development set we provide", is it mean that we are allowed to use **only** the scene provided in the ARKitScenes and the 3DOD annotations? 
Can we use open-source pretrained models such as [OpenScene](https://pengsongyou.github.io/openscene) trained on [Scannetv2 ](http://www.scan-net.org/)or [CLIP](https://github.com/openai/CLIP), Image backbones pretrained on ImageNet etc.? If yes, are we allowed to use additional indoor datasets such as Scannetv2, [Matterport3D](https://niessner.github.io/Matterport/), etc.? We don't quite sure about the challenge regulation (dataset, pretrained model usage). Please carify about the challenge regulations more.

Looking forward to the respone.
Thanks you,
PhucNDA
